presidio-anonymizer/presidio_anonymizer/entities/analyzer_result.py

```diff
@@ -109,5 +116,6 @@ def validate_and_create(cls, content):
     def __validate_fields(cls, content):
         for field in ("start", "end", "score", "entity_type"):
             if content.get(field) is None:
+                cls.logger.debug(f"invalid input, no field {field} for {content}")
```

Comment:
just curios, any particular reason to define it in the class? unlike other cases?

Reply:
Done.